packages/core/test/setup.types.test.ts

```diff
@@ -2043,6 +2045,8 @@ describe('setup()', () => {
         a: {
           after: {
             // @x-ts-expect-error https://github.com/microsoft/TypeScript/issues/55709
+            // TypeScript limitation: mapped types with string unions don't narrow properly.
+            // A dev-mode runtime warning will be shown for this.
```

Comment:
These comments are unrelated to any of the changes made in this PR and shouldn't be committed.